docs/cheaha/tutorial/pytorch_tensorflow.md

```diff
@@ -0,0 +1,133 @@
+# Anaconda Environment Tutorial for PyTorch and TensorFlow
+
+The below tutorial would show you steps on how to create an Anaconda environment, activate, and install libraries/packages for machine and deep learning (PyTorch and Tensorflow) using an Anaconda environment on Cheaha. There are also steps on how to access and use the Shell Command Line Interface (CLI), and using Jupyter Notebook's Graphical User Interface (GUI) to work with these Anaconda environments. There are detailed steps here to guide your creation of a [Jupyter Notebook job.](../open_ondemand/ood_layout.md#interactive-apps)
+
+## Installing Anaconda Environments using Cheaha's Command Line Interface (CLI)
```

Comment:
Please shorten all the headings where possible.

This one could be "Starting a Jupyter Notebook job."

Reply:
Made this change